Tests/Factory/HttpFoundationFactoryTest.php

```diff
@@ -174,15 +174,15 @@ public function testCreateUploadedFileWithError()
         $symfonyUploadedFile->move($this->tmpDir, 'shouldFail.txt');
     }
 
-    private function createUploadedFile($content, $error, $clientFileName, $clientMediaType)
+    private function createUploadedFile($content, $error, $clientFileName, $clientMediaType): UploadedFile
     {
         $filePath = tempnam($this->tmpDir, uniqid());
         file_put_contents($filePath, $content);
 
         return new UploadedFile($filePath, filesize($filePath), $error, $clientFileName, $clientMediaType);
     }
 
-    private function callCreateUploadedFile(UploadedFileInterface $uploadedFile)
+    private function callCreateUploadedFile(UploadedFileInterface $uploadedFile): \Symfony\Component\HttpFoundation\File\UploadedFile
```

Comment:
should use the `use`

Reply:
Since there was already a use for UploadedFile (from `Symfony\Bridge\PsrHttpMessage\Tests\Fixtures`) I fixed this comment with `Symfony\Component\HttpFoundation\File\UploadedFile as HttpFoundationUploadedFile` -- hope that's ok.